app/javascript/packs/stylesheets/admin.scss

```diff
@@ -0,0 +1,3 @@
+#admin-table-header {
```

Comment:
any reason this was made as an `id` instead of a class?

Reply:
because it's specifically for that admin table header and figured it would be used as an identifier when tests are eventually written

Reply:
Okay fair point about future tests. The rest of the PR looks good to me!